ts/utils/convertDateToWordDistance.ts

```diff
@@ -0,0 +1,21 @@
+import { differenceInCalendarDays, format } from "date-fns";
+import I18n from "../i18n";
+/**
+ * This function converts the distance from now to date in : H.mm, yesterday, D/MM/YY and DD/MM
+ */
+export function convertDateToWordDistance(date: Date): string {
+  const distance = differenceInCalendarDays(new Date(), new Date(date));
+  // 0 days, distance < 24h
```

Comment:
differenceInCalendarDays in not based on 24h.

today@23:59 has a distance < 24h from tomorrow@00:01 but the calendarDays distance === 1

Reply:
good catch!

Reply:
dealing with time is HARD

Reply:
also, I'm not sure what happens when the phone's timezone is different than the message timezone :)

Reply:
The problem i reported was anly for the comment, the function works because it is using differenceInCalendarDays and not differenceInDays (24h version). 
For the timezone i think we need to convert the "new Data(date)" to the same timezone of the phone. momentjs has http://momentjs.com/timezone/docs/ to convert to a specific timezone there is also this function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset

Reply:
could you open a story to track that matter? we're not going to deal with it now...

Reply:
https://www.pivotaltracker.com/story/show/158389284

Reply:
Fixed the comment, and add a TODO with pivotal id.